telco-hub/configuration/reference-crs/required/gitops/kustomization.yaml

```diff
@@ -5,9 +5,15 @@ resources:
   - app-project.yaml
   - argocd-application.yaml
   - argocd-ssh-known-hosts-cm.yaml
-  - clusterrole.yaml
-  - clusterrolebinding.yaml
-  - gitopsNS.yaml
-  - gitopsOperatorGroup.yaml
-  - gitopsSubscription.yaml
```

Comment:
These are all required in order for the operator to apply the CRs in this reference. Although they are created when the user first applies this project/application, having them here as well ensures that the configuration is tracked/maintained in git going forward.